sdk/identity/azure-identity/src/main/java/com/azure/identity/implementation/IdentityClientOptions.java

```diff
@@ -781,6 +782,16 @@ public IdentityClientOptions setEnableLegacyMsaPassthrough(boolean enableMsaPass
         return this;
     }
 
+    /**
+     * Sets whether to use the logged-in user's account for broker authentication.
+     * @param useOperatingSystemAccount
+     * @return the updated client options
+     */
+    public IdentityClientOptions setUseOperatingSystemAccount(boolean useOperatingSystemAccount) {
```

Comment:
@christothes is this the name we've picked for all languages

Reply:
Answered my own question --- https://github.com/Azure/azure-sdk-for-net/pull/39607/files

Reply:
worth noting this is the internal property bag. The public API is `useOperatingSystemAccount` on the [builder here](https://github.com/Azure/azure-sdk-for-java/pull/38792/files#diff-6bf12785e338eb151b9328776f5ee2dd11623f7028c0873edc1910105b58bbebR70).